### PR TITLE
OCPQE-24729 Refine eus upgrade jobs for 4.14 to 4.16 upgrade path

### PR DESCRIPTION
**Pull request by JianLi-RH**
/cc @jiajliu @jianlinliu @shellyyang1989

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-eus-4.14.yaml [hunk 1]
```diff
@@ -60,66 +60,10 @@ resources:
...
-    - chain: openshift-upgrade-qe-test-eus
-    workflow: cucushift-installer-rehearse-gcp-ipi-workers-rhel8-fips
-- as: ibmcloud-ipi-f14
-  cron: 31 19 7,23 * *
+- as: ibmcloud-ipi-ovn-ipsec-f14
+  cron: 12 18 6,20 * *
   steps:
     cluster_profile: ibmcloud-qe
-    env:
-      FILTERS_ADDITIONAL: ~TELCO&
```
Comment (jiajliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-eus-4.14.yaml:118]: we need the filter for eus upgrade.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__arm64-nightly-4.16-upgrade-from-eus-4.14.yaml [hunk 2]
```diff
@@ -77,22 +77,17 @@ resources:
...
   steps:
-    cluster_profile: aws-qe
+    cluster_profile: equinix-ocp-metal-qe
     dependencies:
       OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:latest-arm64
     env:
-      BASE_DOMAIN: qe.devcluster.openshift.com
-      COMPUTE_NODE_TYPE: m6g.xlarge
-      CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
-      FILTERS_ADDITIONAL: ~TELCO&
```
Comment (jiajliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__arm64-nightly-4.16-upgrade-from-eus-4.14.yaml:90]: the same here

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__arm64-nightly-4.16-upgrade-from-eus-4.14.yaml [hunk 3]
```diff
@@ -108,22 +103,16 @@ tests:
...
+  cron: 35 21 5,21 * *
   steps:
-    cluster_profile: azure-qe
+    cluster_profile: gcp-qe
     dependencies:
       OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:latest-arm64
     env:
-      BASE_DOMAIN: qe.azure.devcluster.openshift.com
-      COMPUTE_NODE_TYPE: Standard_D4ps_v5
-      FILTERS_ADDITIONAL: ~TELCO&
```
Comment (jiajliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__arm64-nightly-4.16-upgrade-from-eus-4.14.yaml:121]: and this one

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly-4.16-upgrade-from-eus-4.14.yaml [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: aws-ipi-ovn-ipsec-arm-mixarch-f14
+  cron: 6 23 11,25 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
```
Comment (jiajliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly-4.16-upgrade-from-eus-4.14.yaml:67]: `FILTERS_ADDITIONAL: ~TELCO&`

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly-4.16-upgrade-from-eus-4.14.yaml [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+          cpu: 100m
+          memory: 200Mi
+    - ref: ipi-install-heterogeneous
+    - chain: openshift-upgrade-qe-test-eus
+    workflow: cucushift-installer-rehearse-aws-ipi-ovn-ipsec
+- as: azure-ipi-arm-mixarch-f14
+  cron: 11 2 5,19 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
```
Comment (jiajliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly-4.16-upgrade-from-eus-4.14.yaml:96]: the same with https://github.com/openshift/release/pull/56164/files#r1740376884

**Comment by jiajliu** [comment]
lgtm

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-eus-4.14.yaml [hunk 6]
```diff
@@ -134,17 +80,15 @@ tests:
           cpu: 100m
           memory: 200Mi
     - chain: openshift-upgrade-qe-test-eus
-    workflow: cucushift-installer-rehearse-ibmcloud-ipi
-- as: nutanix-ipi-proxy-fips-f28
+    workflow: cucushift-installer-rehearse-ibmcloud-ipi-ovn-ipsec
+- as: nutanix-ipi-proxy-fips-f14
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-eus-4.14.yaml:84]: Nutanix is a small deployment, we can not bear many concurrent running cluster, I suggest to keep the original `f28`

Comment (JianLi-RH) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-eus-4.14.yaml:84]: I will also update baremetalds-ipi-ovn-ipv4-f14 to f28 in openshift-openshift-tests-private-release-4.16__arm64-nightly-4.16-upgrade-from-eus-4.14.yaml

Comment (JianLi-RH) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-eus-4.14.yaml]: done

**Pull request merged**